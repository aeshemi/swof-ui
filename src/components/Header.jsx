import React from 'react';
import { Nav, Navbar, NavItem } from 'react-bootstrap';
import { LinkContainer } from 'react-router-bootstrap';

const Header = () => (
  <Navbar inverse>
    <Navbar.Header>
      <Navbar.Brand>Support Wheel of Fate</Navbar.Brand>
      <Navbar.Toggle />
    </Navbar.Header>
    <Navbar.Collapse>
      <Nav pullRight>
        <LinkContainer to="/" exact>
          <NavItem>Calendar</NavItem>
        </LinkContainer>
        <LinkContainer to="/list">
          <NavItem>List</NavItem>
        </LinkContainer>
        <LinkContainer to="/generator">
          <NavItem>Generator</NavItem>
        </LinkContainer>
      </Nav>
    </Navbar.Collapse>
  </Navbar>
);

export default Header;
