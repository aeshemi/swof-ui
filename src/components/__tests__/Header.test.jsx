import Header from '../Header';

it('NotFound renders correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});
