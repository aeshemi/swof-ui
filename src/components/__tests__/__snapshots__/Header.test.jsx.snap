// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<Uncontrolled(Navbar)
  bsStyle="default"
  inverse={true}
>
  <NavbarHeader>
    <NavbarBrand>
      Support Wheel of Fate
    </NavbarBrand>
    <NavbarToggle />
  </NavbarHeader>
  <NavbarCollapse>
    <Nav
      bsClass="nav"
      justified={false}
      pullLeft={false}
      pullRight={true}
      stacked={false}
    >
      <LinkContainer
        activeClassName="active"
        exact={true}
        replace={false}
        strict={false}
        to="/"
      >
        <NavItem
          active={false}
          disabled={false}
        >
          Calendar
        </NavItem>
      </LinkContainer>
      <LinkContainer
        activeClassName="active"
        exact={false}
        replace={false}
        strict={false}
        to="/list"
      >
        <NavItem
          active={false}
          disabled={false}
        >
          List
        </NavItem>
      </LinkContainer>
      <LinkContainer
        activeClassName="active"
        exact={false}
        replace={false}
        strict={false}
        to="/generator"
      >
        <NavItem
          active={false}
          disabled={false}
        >
          Generator
        </NavItem>
      </LinkContainer>
    </Nav>
  </NavbarCollapse>
</Uncontrolled(Navbar)>
`;
